pkg/apis/wing/types.go

```diff
@@ -52,7 +52,8 @@ type InstanceStatusManifest struct {
 
 type InstanceList struct {
 	metav1.TypeMeta
-	metav1.ObjectMeta
+	// +optional
+	metav1.ListMeta
```

Comment:
Don't think this is, or should be, optional?

Reply:
Just copied from the core:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L3409